components/cache/cache_pools.rst

```diff
@@ -58,6 +58,25 @@ contents as regular files in a set of directories on the local file system::
         $directory = null
     );
 
+Php Files Cache Adapter
```

Comment:
I'm going to leave this - I did it because the name of the class is PhpFilesAdapter... but I could see it either way (I would normally 100% agree)